assets/styles/_base.scss

```
@@ -33,6 +33,10 @@ html {
 body {
   height: 100%;
   overflow-x: hidden;
+
+  @media (min-width: $breakpoint-mobile-horizontal) {
+    font-size: 1.13rem;
```

This translates to 18.08px, it would be great ending up with an even number of 18px instead.
```suggestion
    font-size: 1.125rem;
```

Additionally, I would be curios if we can use that size on mobile as well!

---

`In general, the rule of thumb is that font size needs to be 16 pixels for mobile websites. Anything smaller than that could compromise readability for visually impaired readers. Anything too much larger could also make reading more difficult. You want to find that perfect Goldilocks formula and, time and time again, it comes back to 16 pixels.`

I agree with this statement, I like how 16px looks on mobile. 18px looks too big for me

---

Alright, thanks for researching this topic!